views/components/Certification.js

```diff
@@ -1,72 +1,418 @@
+            <Develop
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+            <Deploy
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+            <Deliver
+              itemSelected={itemSelected}
+              setSelectedItem={setSelectedItem}
+            />
+          </List>
+        </div>
       </div>
-    </div>
-  </section>
-)
+    </Container>
+  )
+}
+
+const Research = ({ itemSelected, setSelectedItem }) => {
+  return (
+    <>
+      <ResearchItem onClick={() => setSelectedItem(INIT_CERT_ACTIVE_ITEM[0])}>
+        <ItemIcon1 id={INIT_CERT_ACTIVE_ITEM[0]} itemSelected={itemSelected} />
+      </ResearchItem>
+
+      <ResearchInfo itemSelected={itemSelected}>
+        <Header>
+          <span>1</span> &mdash; Bachelor of Science
+        </Header>
+        <p>
+          Graduating from HCM University of Science, Advanced Program in
+          Computer Science.
+        </p>
+      </ResearchInfo>
+    </>
+  )
+}
+
+const Develop = ({ itemSelected, setSelectedItem }) => {
+  return (
+    <>
+      <DevelopItem onClick={() => setSelectedItem(INIT_CERT_ACTIVE_ITEM[1])}>
+        <ItemIcon2 id={INIT_CERT_ACTIVE_ITEM[1]} itemSelected={itemSelected} />
+      </DevelopItem>
+
+      <DevelopInfo itemSelected={itemSelected}>
+        <Header>
+          <span>2</span> &mdash; Communicative Certification: TOEFL ibt 85 pts
+        </Header>
+        <p>
+          The TOEFL iBT® test measures your ability to use and understand
+          English at the university level. And it evaluates how well you combine
+          your reading, listening, speaking and writing skills to perform
+          academic tasks.
+        </p>
+      </DevelopInfo>
+    </>
+  )
+}
+
+const Deploy = ({ itemSelected, setSelectedItem }) => {
+  return (
+    <>
+      <DeployItem onClick={() => setSelectedItem(INIT_CERT_ACTIVE_ITEM[2])}>
+        <ItemIcon3 id={INIT_CERT_ACTIVE_ITEM[2]} itemSelected={itemSelected} />
+      </DeployItem>
+
+      <DeployInfo itemSelected={itemSelected}>
+        <Header>
+          <span>3</span> &mdash; Second place
+        </Header>
+        <p>
+          Got second prize for the competitive algorithm contest of the
+          university
+        </p>
+      </DeployInfo>
+    </>
+  )
+}
+
+const Deliver = ({ itemSelected, setSelectedItem }) => {
+  return (
+    <>
+      <DeliverItem onClick={() => setSelectedItem(INIT_CERT_ACTIVE_ITEM[3])}>
+        <ItemIcon4 id={INIT_CERT_ACTIVE_ITEM[3]} itemSelected={itemSelected} />
+      </DeliverItem>
+
+      <DeliverInfo itemSelected={itemSelected}>
+        <Header>
+          <span>4</span> &mdash; First place
+        </Header>
+        <p>
+          Highschool, got First prize for the competitive algorithm contest of
+          the city.
+        </p>
+      </DeliverInfo>
+    </>
+  )
+}
+
+const ItemIcon1 = ({ id, itemSelected }) => {
```

Comment:
We should merge 4 Icon into one ItemIcon with extra param `name`

Reply:
Updated